src/iap/modules/dev/components/dev-page/dev-page.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+
+import { PreviewSlideComponent } from '@shared/components';
+
+import { Slide } from '@shared/models/carousel.models';
+import { Game } from '@shared/models/dto/game.model';
```

Comment:
:point_right: :point_left: 
```suggestion
import { PreviewSlideComponent } from '@shared/components';
import { Slide } from '@shared/models/carousel.models';
import { Game } from '@shared/models/dto/game.model';
```